site/class-1-load/Senpaizuri/index.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<meta charset=utf8>
+<title>@Senpaizuri</title>
+<meta content=width=device-width,initial-scale=1 name='viewport'>
+<script src="https://d3js.org/d3.v4.min.js"></script>
```

Comment:
You’re linking to d3 here, which is great, but you’ve also included d3 in Git.
Could you remove the `d3.js` file?